src/styles/blocks/modifiers.scss

```diff
@@ -0,0 +1,18 @@
+@for $day-number from 1 through 31 {
+  .calendar__day:nth-child(#{$day-number})::before {
+    content: '#{$day-number}';
+    color: #000;
+    padding-left: 1px;
+  }
```

Comment:
You should not hardcode the color value here. It would be better to define this color as a variable in the `variables.scss` file and use it here. This will make it easier to change the color scheme of your calendar in the future.